src/app/lists/CommentBox.tsx

```diff
@@ -0,0 +1,128 @@
+"use client";
+
+import FreeFormTextInput from "@/components/DataInput/FreeFormTextInput";
+import supabase from "@/supabase";
+import Button from "@mui/material/Button";
+import React, { useState } from "react";
+import styled from "styled-components";
+
+interface CommentProps {
+    originalComment: string;
+}
+
+const HeaderAndButtonContainer = styled.div`
+    display: flex;
+    align-items: center;
+    width: 100%;
+    min-width: 8rem;
+    flex-direction: column;
+    gap: 1rem;
+    justify-content: space-between;
+    @media (min-width: 800px) {
+        width: 15%;
```

Comment:
does this look wildly different at 799px and 801px?

Reply:
![799px - snip](https://github.com/VauxhallFoodbank/vauxhall-foodbank/assets/137791712/44e78f65-9c83-4f24-b68c-264cdfa2675b)
![800px+ snip](https://github.com/VauxhallFoodbank/vauxhall-foodbank/assets/137791712/0d88f20e-d416-462e-9960-5ed5e0396256)

Reply:
Bottom one is 800px+

Reply:
Needs to be different as otherwise it looks odd when the comment box is really short

Reply:
this looks great!